src/celldega/pre/image_quantification.py

```diff
@@ -0,0 +1,339 @@
+import os
+import glob
+import zarr
+import tifffile
+import pandas as pd
+import numpy as np
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import rasterio
+from rasterio.mask import mask
+from rasterstats import zonal_stats
```

Comment:
I believe `zontal_stats` is never used, can you remove it along other functions that not being used in this file? Thx!

Reply:
You probably want to include `rasterio` in the toml file if we decide to use it.